### PR TITLE
Rename MainActivity files, add test cases to set plugins, and fix to copy build folders

### PR DESCRIPTION
**Pull request by ElenaDiachenko**
## Description

- Rename MainActivity_java to MainActivity_kt 
- add a test case  to test AndroidManifest_xml overrides in app-harness ( "react-native-photo-editor"  plugin)
- fix copying of builds folder from external pluginTemplates
- add a test case to test the override of builds folder from an external package
- allow the androidManifest attribute to be set to `false`:
```
 "platforms": {
        "android": {
            "templateAndroid": {
                "AndroidManifest_xml": {
                    "tag": "manifest",
                    "android:name": "",
                    "children": [
                        {
                            "tag": "application",
                            "android:name": ".MainApplication",
                            "android:largeHeap": false,
                            "children": []
                        }
                    ]
                }
            }
        }
    },
```

## Related issues

n/a

## Npm releases

n/a

NOTE:
test this PR following following: 
- https://github.com/flexn-io/renative/pull/1454

### CODE REVIEW

**Comment by pauliusguzas** [comment]
@ElenaDiachenko any way to test this?

**Comment by ElenaDiachenko** [comment]
@pauliusguzas 
Check https://github.com/flexn-io/renative/issues/1435. I added settings to `app-harness/renative.json/plugins/"react-native-photo-editor"/android/templateAndroid/AndroidManifest_xml`. 

If you run android and check `platformBuilds/harness_android/app/src/main/AndroidManifest.xml `you will see these activities and uses-permission being injected.

**Comment by pauliusguzas** [comment]
@ElenaDiachenko I think we agreed that no testing will be done via code, test case on app-harness maybe should be added. cc @pavjacko @mihaiblaga89

File: packages/sdk-android/src/kotlinParser.ts [hunk 1]
```diff
@@ -16,32 +16,32 @@ import { getBuildFilePath, getEntryFile, getAppId, addSystemInjects } from '@rnv
 //     androidwear: '"assets://index.androidwear.bundle"',
 // };
 
-export const parseFlipperSync = (c: Context, scheme: 'debug' | 'release') => {
-    const appFolder = getAppFolder(c);
-    const { platform } = c;
+// export const parseFlipperSync = (c: Context, scheme: 'debug' | 'release') => {
```
Comment (mihaiblaga89) [inline packages/sdk-android/src/kotlinParser.ts:19]: @ElenaDiachenko please remove the commented out code

**Review by mihaiblaga89 — CHANGES_REQUESTED** [review]

File: packages/sdk-android/src/kotlinParser.ts (hunk 1 above)
Comment (pavjacko) [reply packages/sdk-android/src/kotlinParser.ts:19]: @ElenaDiachenko why has this been commented out? are we not supporting flipper ?

File: packages/app-harness/renative.json [hunk 2]
```diff
@@ -105,7 +110,8 @@
                                         "tag": "activity",
                                         "android:name": "com.ahmedadeltito.photoeditor.PhotoEditorActivity",
                                         "children": []
-                                    }
+                                    },
+                                    { "tag": "activity", "android:name": "com.yalantis.ucrop.UCropActivity" }
```
Comment (pavjacko) [inline packages/app-harness/renative.json:114]: @ElenaDiachenko so does this change mean you also fixing something else apart from just pure renaming of activity? if so can you reference issue or describe it in the PR

File: packages/sdk-android/src/manifestParser.ts [hunk 3]
```diff
@@ -114,7 +114,7 @@ const _mergeNodeParameters = (
         const key = k as NodeKey;
         const val = nodeParamsExt[key];
 
-        if (val && !SYSTEM_TAGS.includes(k)) {
+        if (val !== 'undefined' && !SYSTEM_TAGS.includes(k)) {
```
Comment (pavjacko) [inline packages/sdk-android/src/manifestParser.ts:117]: @ElenaDiachenko can you clarify in description what does this line fixes. is it reproducible for @pauliusguzas @GabrieleKaceviciute ?

**Review by pavjacko — CHANGES_REQUESTED** [review]

**Review by GabrieleKaceviciute — CHANGES_REQUESTED** [review]
can't run app-harness on `androidwear` and `androidtv` ->
```
app-harness git:(fix/androidManifest_generating) npx rnv run -p androidwear

┌─────────────────────────────────────────────────────────────────┐
│ ██████╗ ███████╗███╗   ██╗ █████╗ ████████╗██╗██╗   ██╗███████╗ │
│ ██╔══██╗██╔════╝████╗  ██║██╔══██╗╚══██╔══╝██║██║   ██║██╔════╝ │
│ ██████╔╝█████╗  ██╔██╗ ██║███████║   ██║   ██║██║   ██║█████╗   │
│ ██╔══██╗██╔══╝  ██║╚██╗██║██╔══██║   ██║   ██║╚██╗ ██╔╝██╔══╝   │
│ ██║  ██║███████╗██║ ╚████║██║  ██║   ██║   ██║ ╚████╔╝ ███████╗ │
│ ╚═╝  ╚═╝╚══════╝╚═╝  ╚═══╝╚═╝  ╚═╝   ╚═╝   ╚═╝  ╚═══╝  ╚══════╝ │
│ 🚀 v:1.0.0-rc.12 | renative.org | 3/14/2024, 6:07:53 PM         │
│ $ npx rnv run -p androidwear                                    │
└─────────────────────────────────────────────────────────────────┘
info: Load engine: @rnv/engine-rn (./node_modules/@rnv/engine-rn)
info: Load engine: @rnv/engine-rn-tvos (./node_modules/@rnv/engine-rn-tvos)
info: Load engine: @rnv/engine-rn-next (./node_modules/@rnv/engine-rn-next)
info: Load engine: @rnv/engine-rn-web (./node_modules/@rnv/engine-rn-web)
info: Load engine: @rnv/engine-rn-electron (./node_modules/@rnv/engine-rn-electron)
info: Current engine: engine-rn (../engine-rn)
task: ○ run [1]
task: ○ configure [1]
task: ○ platform configure [1]
task: ○ project configure [1]
task: ○ workspace configure [1]
task: ✔ workspace configure [1]
task: ○ install [1]
task: ✔ install [1]
task: ○ install [2]
task: ✔ install [2]
task: ○ app configure [1]
info: Current app config: harness
task: ✔ app configure [1]
info: Plugin overrides disabled for: react-native-gesture-handler with disablePluginTemplateOverrides. SKIPPING
info: ../../node_modules/@react-native-community/cli-platform-ios/build/tools/listIOSDevices.js overriden by: ../rnv/pluginTemplates/@react-native-community/cli-platform-ios/overrides@11.3.7.json
info: Plugin overrides disabled for: next with disablePluginTemplateOverrides. SKIPPING
info: ../../node_modules/rn-fetch-blob overriden by: /@flexn/plugins/pluginTemplates/rn-fetch-blob/overrides
info: ../../node_modules/react-native-photo-editor overriden by: /@flexn/plugins/pluginTemplates/react-native-photo-editor/overrides
info: Plugin overrides disabled for: react-native-permissions with disablePluginTemplateOverrides. SKIPPING
task: ✔ project configure [1]
info: Current Build Scheme: debug
task: ○ install [3]
task: ✔ install [3]
task: ✔ platform configure [1]
info: Found custom assetFolderPlatform: android. Will use it instead of deafult androidwear
info: Found custom assetSources at ../../node_modules/@flexn/assets-renative-outline/assets/android. Will be used to generate assets.
task: ✔ configure [1]
✔ Executing: /Users/gabriele/Library/Android/sdk/platform-tools/adb devices -l
✔ Executing: /Users/gabriele/Library/Android/sdk/emulator/emulator -list-avds
✔ Waiting for emulator to boot...
✔ Executing: /Users/gabriele/Library/Android/sdk/platform-tools/adb -s emulator-5554 shell wm size
✔ Executing: /Users/gabriele/Library/Android/sdk/platform-tools/adb -s emulator-5554 shell wm density
✔ Executing: /Users/gabriele/Library/Android/sdk/platform-tools/adb -s emulator-5554 shell getprop
task: ○ start [1]

Dev server running at: http://0.0.0.0:8089/index.bundle?platform=android

task: ✔ start [1]
⠼ Waiting for bundler to finish...
  WARNING: You should run npx react-native@latest to ensure you're always using the most current version of the CLI. NPX has cached version (0.73.4) != current release (0.73.6)

⠏ Waiting for bundler to finish...warn Package rn-fetch-blob contains invalid configuration: "dependency.hooks" is not allowed. Please verify it's properly linked using "npx react-native config" command and contact the package maintainers about this.
warn =================================================================================================
warn From React Native 0.73, your project's Metro config should extend '@react-native/metro-config'
warn or it will fail to build. Please copy the template at:
warn https://github.com/facebook/react-native/blob/main/packages/react-native/template/metro.config.js
warn This warning will be removed in future (https://github.com/facebook/metro/issues/1018).
warn =================================================================================================
info Welcome to React Native v0.73
info Starting dev server on port 8089...

                        ▒▒▓▓▓▓▒▒
                     ▒▓▓▓▒▒░░▒▒▓▓▓▒
                  ▒▓▓▓▓░░░▒▒▒▒░░░▓▓▓▓▒
                 ▓▓▒▒▒▓▓▓▓▓▓▓▓▓▓▓▓▒▒▒▓▓
                 ▓▓░░░░░▒▓▓▓▓▓▓▒░░░░░▓▓
                 ▓▓░░▓▓▒░░░▒▒░░░▒▓▒░░▓▓
                 ▓▓░░▓▓▓▓▓▒▒▒▒▓▓▓▓▒░░▓▓
                 ▓▓░░▓▓▓▓▓▓▓▓▓▓▓▓▓▒░░▓▓
                 ▓▓▒░░▒▒▓▓▓▓▓▓▓▓▒░░░▒▓▓
                  ▒▓▓▓▒░░░▒▓▓▒░░░▒▓▓▓▒
                     ▒▓▓▓▒░░░░▒▓▓▓▒
                        ▒▒▓▓▓▓▒▒


                Welcome to Metro v0.80.5
              Fast - Scalable - Integrated


⠙ Waiting for bundler to finish...info React Native v0.73.6 is now available (your project is running on v0.73.4).
info Changelog: https://github.com/facebook/react-native/releases/tag/v0.73.6
info Diff: https://react-native-community.github.io/upgrade-helper/?from=0.73.6
info For more info, check out "https://reactnative.dev/docs/upgrading?os=macos".
✔ Waiting for bundler to finish...
info: ANDROID PACKAGE STARTING...
✔ Executing: react-native bundle --platform android --dev false --assets-dest /Users/gabriele/Desktop/tests/renative/packages/app-harness/platformBuilds/harness_androidwear/app/src/main/res --entry-file index.js --bundle-output /Users/gabriele/Desktop/tests/renative/packages/app-harness/platformBuilds/harness_androidwear/app/src/main/assets/index.bundle --config=metro.config.js
info: ANDROID PACKAGE FINISHED

  WARNING: You should run npx react-native@latest to ensure you're always using the most current version of the CLI. NPX has cached version (0.73.4) != current release (0.73.6)

warn Package rn-fetch-blob contains invalid configuration: "dependency.hooks" is not allowed. Please verify it's properly linked using "npx react-native config" command and contact the package maintainers about this.
info Building the app...

> Configure project :react-native-firebase_app
:react-native-firebase_app package.json found at /Users/gabriele/Desktop/tests/renative/node_modules/@react-native-firebase/app/package.json
:react-native-firebase_app:firebase.bom using default value: 32.5.0
:react-native-firebase_app:play.play-services-auth using default value: 20.7.0
:react-native-firebase_app package.json found at /Users/gabriele/Desktop/tests/renative/node_modules/@react-native-firebase/app/package.json
:react-native-firebase_app:version set from package.json: 18.6.1 (18,6,1 - 18006001)
:react-native-firebase_app:android.compileSdk using custom value: 34
:react-native-firebase_app:android.targetSdk using custom value: 34
:react-native-firebase_app:android.minSdk using custom value: 26
:react-native-firebase_app:reactNativeAndroidDir /Users/gabriele/Desktop/tests/renative/node_modules/react-native/android
WARNING: The specified Android SDK Build Tools version (28.0.2) is ignored, as it is below the minimum supported version (33.0.1) for Android Gradle Plugin 8.1.1.
Android SDK Build Tools 33.0.1 will be used.
To suppress this warning, remove "buildToolsVersion '28.0.2'" from your build.gradle file, as each version of the Android Gradle Plugin now has a default version of the build tools.

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':react-native-photo-editor:bundleLibCompileToJarDebug'.
> Could not create task ':react-native-photo-editor:compileDebugJavaWithJavac'.
   > In order to compile Java 9+ source, please set compileSdkVersion to 30 or above

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.3/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
5 actionable tasks: 5 up-to-date

info 💡 Tip: Make sure that you have set up your development environment correctly, by running npx react-native doctor. To read more about doctor command visit: https://github.com/react-native-community/cli/blob/main/packages/cli-doctor/README.md#doctor

error Failed to build the app.
Error: Command failed with exit code 1: ./gradlew app:installDebug -x lint -PreactNativeDevServerPort=8089
    at makeError (/Users/gabriele/Desktop/tests/renative/node_modules/execa/lib/error.js:60:11)
    at Function.module.exports.sync (/Users/gabriele/Desktop/tests/renative/node_modules/execa/index.js:194:17)
    at build (/Users/gabriele/Desktop/tests/renative/node_modules/react-native-tvos/node_modules/@react-native-community/cli-platform-android/build/commands/buildAndroid/index.js:68:22)
    at runOnSpecificDevice (/Users/gabriele/Desktop/tests/renative/node_modules/react-native-tvos/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/index.js:220:33)
    at buildAndRun (/Users/gabriele/Desktop/tests/renative/node_modules/react-native-tvos/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/index.js:182:12)
    at Object.runAndroid [as func] (/Users/gabriele/Desktop/tests/renative/node_modules/react-native-tvos/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/index.js:115:10)
    at Command.handleAction (/Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/@react-native-community/cli/build/index.js:118:23)
    at Command.listener [as _actionHandler] (/Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:482:17)
    at /Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:1283:65
    at Command._chainOrCall (/Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:1177:12)
    at Command._parseCommand (/Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:1283:27)
    at /Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:1081:27
    at Command._chainOrCall (/Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:1177:12)
    at Command._dispatchSubcommand (/Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:1077:23)
    at Command._parseCommand (/Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:1248:19)
    at Command.parse (/Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:909:10)
┌─────────────────────────────────────────────────────────────────┐
│ ✔ SUMMARY | 3/14/2024, 6:08:23 PM | rnv@1.0.0-rc.12             │
│ $ npx rnv run -p androidwear                                    │
├─────────────────────────────────────────────────────────────────┤
│ Project: @rnv/app-harness@1.0.0-rc.12                           │
│ Workspace: rnv                                                  │
│ Platform (-p): androidwear                                      │
│ Engine: engine-rn                                               │
│ Template: @rnv/template-starter@1.0.0-rc.12                     │
│ App Config (-c): harness                                        │
│ Build Scheme (-s): debug                                        │
│ Target (-t): Wear_OS_Square_API_30                              │
│ Supported Platforms: androidwear                                │
│ Env Info: darwin | arm64 | node v18.19.0                        │
│ Executed Time: 0h:0m:30s:62ms                                   │
│ error: ⨯ ○ run: COMMAND:

npx react-native run-android --mode=Debug --no-packager --main-activity=MainActivity --deviceId=emulator-5554

FAILED with ERROR:

Error: Command failed with exit code 1: npx react-native run-android --mode=Debug --no-packager --main-activity=MainActivity --deviceId=emulator-5554
    at makeError (/Users/gabriele/Desktop/tests/renative/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/Users/gabriele/Desktop/tests/renative/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)


│ Project location: ./platformBuilds/harness_androidwear          │
└─────────────────────────────────────────────────────────────────┘
➜  app-harness git:(fix/androidManifest_generating)
```
androidtv ->
```
➜  app-harness git:(fix/androidManifest_generating) npx rnv run -p androidtv

┌─────────────────────────────────────────────────────────────────┐
│ ██████╗ ███████╗███╗   ██╗ █████╗ ████████╗██╗██╗   ██╗███████╗ │
│ ██╔══██╗██╔════╝████╗  ██║██╔══██╗╚══██╔══╝██║██║   ██║██╔════╝ │
│ ██████╔╝█████╗  ██╔██╗ ██║███████║   ██║   ██║██║   ██║█████╗   │
│ ██╔══██╗██╔══╝  ██║╚██╗██║██╔══██║   ██║   ██║╚██╗ ██╔╝██╔══╝   │
│ ██║  ██║███████╗██║ ╚████║██║  ██║   ██║   ██║ ╚████╔╝ ███████╗ │
│ ╚═╝  ╚═╝╚══════╝╚═╝  ╚═══╝╚═╝  ╚═╝   ╚═╝   ╚═╝  ╚═══╝  ╚══════╝ │
│ 🚀 v:1.0.0-rc.12 | renative.org | 3/14/2024, 6:11:21 PM         │
│ $ npx rnv run -p androidtv                                      │
└─────────────────────────────────────────────────────────────────┘
info: Load engine: @rnv/engine-rn (./node_modules/@rnv/engine-rn)
info: Load engine: @rnv/engine-rn-tvos (./node_modules/@rnv/engine-rn-tvos)
info: Load engine: @rnv/engine-rn-next (./node_modules/@rnv/engine-rn-next)
info: Load engine: @rnv/engine-rn-web (./node_modules/@rnv/engine-rn-web)
info: Load engine: @rnv/engine-rn-electron (./node_modules/@rnv/engine-rn-electron)
info: Current engine: engine-rn-tvos (../engine-rn-tvos)
task: ○ run [1]
task: ○ configure [1]
task: ○ platform configure [1]
task: ○ project configure [1]
task: ○ workspace configure [1]
task: ✔ workspace configure [1]
task: ○ install [1]
task: ✔ install [1]
task: ○ install [2]
task: ✔ install [2]
task: ○ app configure [1]
info: Current app config: harness
task: ✔ app configure [1]
info: Plugin overrides disabled for: react-native-gesture-handler with disablePluginTemplateOverrides. SKIPPING
info: ../../node_modules/@react-native-community/cli-platform-ios/build/tools/listIOSDevices.js overriden by: ../rnv/pluginTemplates/@react-native-community/cli-platform-ios/overrides@11.3.7.json
info: Plugin overrides disabled for: next with disablePluginTemplateOverrides. SKIPPING
info: ../../node_modules/rn-fetch-blob overriden by: /@flexn/plugins/pluginTemplates/rn-fetch-blob/overrides
info: ../../node_modules/react-native-photo-editor overriden by: /@flexn/plugins/pluginTemplates/react-native-photo-editor/overrides
info: Plugin overrides disabled for: react-native-permissions with disablePluginTemplateOverrides. SKIPPING
task: ✔ project configure [1]
info: Current Build Scheme: debug
task: ○ install [3]
task: ✔ install [3]
task: ✔ platform configure [1]
info: Found custom assetFolderPlatform: android. Will use it instead of deafult androidtv
info: Found custom assetSources at ../../node_modules/@flexn/assets-renative-outline/assets/android. Will be used to generate assets.
task: ✔ configure [1]
✔ Executing: /Users/gabriele/Library/Android/sdk/platform-tools/adb devices -l
✔ Executing: /Users/gabriele/Library/Android/sdk/emulator/emulator -list-avds
✔ Executing: /Users/gabriele/Library/Android/sdk/platform-tools/adb devices -l
✔ Waiting for emulator to boot...
✔ Executing: /Users/gabriele/Library/Android/sdk/platform-tools/adb -s emulator-5554 shell wm size
✔ Executing: /Users/gabriele/Library/Android/sdk/platform-tools/adb -s emulator-5554 shell wm density
✔ Executing: /Users/gabriele/Library/Android/sdk/platform-tools/adb -s emulator-5554 shell getprop
info: ✔ Found active emulator! emulator-5554. Will use it
task: ○ start [1]

Dev server running at: http://0.0.0.0:8084/index.bundle?platform=android

task: ✔ start [1]
⠙ Waiting for bundler to finish...warn Package rn-fetch-blob contains invalid configuration: "dependency.hooks" is not allowed. Please verify it's properly linked using "npx react-native config" command and contact the package maintainers about this.
warn =================================================================================================
warn From React Native 0.73, your project's Metro config should extend '@react-native/metro-config'
warn or it will fail to build. Please copy the template at:
warn https://github.com/facebook/react-native/blob/main/packages/react-native/template/metro.config.js
warn This warning will be removed in future (https://github.com/facebook/metro/issues/1018).
warn =================================================================================================
info Welcome to React Native v0.73
info Starting dev server on port 8084...

                        ▒▒▓▓▓▓▒▒
                     ▒▓▓▓▒▒░░▒▒▓▓▓▒
                  ▒▓▓▓▓░░░▒▒▒▒░░░▓▓▓▓▒
                 ▓▓▒▒▒▓▓▓▓▓▓▓▓▓▓▓▓▒▒▒▓▓
                 ▓▓░░░░░▒▓▓▓▓▓▓▒░░░░░▓▓
                 ▓▓░░▓▓▒░░░▒▒░░░▒▓▒░░▓▓
                 ▓▓░░▓▓▓▓▓▒▒▒▒▓▓▓▓▒░░▓▓
                 ▓▓░░▓▓▓▓▓▓▓▓▓▓▓▓▓▒░░▓▓
                 ▓▓▒░░▒▒▓▓▓▓▓▓▓▓▒░░░▒▓▓
                  ▒▓▓▓▒░░░▒▓▓▒░░░▒▓▓▓▒
                     ▒▓▓▓▒░░░░▒▓▓▓▒
                        ▒▒▓▓▓▓▒▒


                Welcome to Metro v0.80.5
              Fast - Scalable - Integrated


⠼ Waiting for bundler to finish...info React Native v0.73.6 is now available (your project is running on v0.73.4).
info Changelog: https://github.com/facebook/react-native/releases/tag/v0.73.6
info Diff: https://react-native-community.github.io/upgrade-helper/?from=0.73.6
info For more info, check out "https://reactnative.dev/docs/upgrading?os=macos".
✔ Waiting for bundler to finish...

  WARNING: You should run npx react-native@latest to ensure you're always using the most current version of the CLI. NPX has cached version (0.73.4) != current release (0.73.6)

warn Package rn-fetch-blob contains invalid configuration: "dependency.hooks" is not allowed. Please verify it's properly linked using "npx react-native config" command and contact the package maintainers about this.
info Building the app...

> Configure project :react-native-firebase_app
:react-native-firebase_app package.json found at /Users/gabriele/Desktop/tests/renative/node_modules/@react-native-firebase/app/package.json
:react-native-firebase_app:firebase.bom using default value: 32.5.0
:react-native-firebase_app:play.play-services-auth using default value: 20.7.0
:react-native-firebase_app package.json found at /Users/gabriele/Desktop/tests/renative/node_modules/@react-native-firebase/app/package.json
:react-native-firebase_app:version set from package.json: 18.6.1 (18,6,1 - 18006001)
:react-native-firebase_app:android.compileSdk using custom value: 34
:react-native-firebase_app:android.targetSdk using custom value: 34
:react-native-firebase_app:android.minSdk using custom value: 26
:react-native-firebase_app:reactNativeAndroidDir /Users/gabriele/Desktop/tests/renative/node_modules/react-native/android
WARNING: The specified Android SDK Build Tools version (28.0.2) is ignored, as it is below the minimum supported version (33.0.1) for Android Gradle Plugin 8.1.1.
Android SDK Build Tools 33.0.1 will be used.
To suppress this warning, remove "buildToolsVersion '28.0.2'" from your build.gradle file, as each version of the Android Gradle Plugin now has a default version of the build tools.

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':react-native-photo-editor:bundleLibCompileToJarDebug'.
> Could not create task ':react-native-photo-editor:compileDebugJavaWithJavac'.
   > In order to compile Java 9+ source, please set compileSdkVersion to 30 or above

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 9.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/8.3/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 4s
5 actionable tasks: 5 up-to-date

info 💡 Tip: Make sure that you have set up your development environment correctly, by running npx react-native doctor. To read more about doctor command visit: https://github.com/react-native-community/cli/blob/main/packages/cli-doctor/README.md#doctor

error Failed to build the app.
Error: Command failed with exit code 1: ./gradlew app:installDebug -x lint -PreactNativeDevServerPort=8084
    at makeError (/Users/gabriele/Desktop/tests/renative/node_modules/execa/lib/error.js:60:11)
    at Function.module.exports.sync (/Users/gabriele/Desktop/tests/renative/node_modules/execa/index.js:194:17)
    at build (/Users/gabriele/Desktop/tests/renative/node_modules/react-native-tvos/node_modules/@react-native-community/cli-platform-android/build/commands/buildAndroid/index.js:68:22)
    at runOnSpecificDevice (/Users/gabriele/Desktop/tests/renative/node_modules/react-native-tvos/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/index.js:220:33)
    at buildAndRun (/Users/gabriele/Desktop/tests/renative/node_modules/react-native-tvos/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/index.js:182:12)
    at Object.runAndroid [as func] (/Users/gabriele/Desktop/tests/renative/node_modules/react-native-tvos/node_modules/@react-native-community/cli-platform-android/build/commands/runAndroid/index.js:115:10)
    at Command.handleAction (/Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/@react-native-community/cli/build/index.js:118:23)
    at Command.listener [as _actionHandler] (/Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:482:17)
    at /Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:1283:65
    at Command._chainOrCall (/Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:1177:12)
    at Command._parseCommand (/Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:1283:27)
    at /Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:1081:27
    at Command._chainOrCall (/Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:1177:12)
    at Command._dispatchSubcommand (/Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:1077:23)
    at Command._parseCommand (/Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:1248:19)
    at Command.parse (/Users/gabriele/Desktop/tests/renative/node_modules/react-native/node_modules/commander/lib/command.js:909:10)
┌─────────────────────────────────────────────────────────────────┐
│ ✔ SUMMARY | 3/14/2024, 6:11:40 PM | rnv@1.0.0-rc.12             │
│ $ npx rnv run -p androidtv                                      │
├─────────────────────────────────────────────────────────────────┤
│ Project: @rnv/app-harness@1.0.0-rc.12                           │
│ Workspace: rnv                                                  │
│ Platform (-p): androidtv                                        │
│ Engine: engine-rn-tvos                                          │
│ Template: @rnv/template-starter@1.0.0-rc.12                     │
│ App Config (-c): harness                                        │
│ Build Scheme (-s): debug                                        │
│ Target (-t): Television_1080p_API_31                            │
│ Supported Platforms: androidtv                                  │
│ Env Info: darwin | arm64 | node v18.19.0                        │
│ Executed Time: 0h:0m:19s:101ms                                  │
│ info: ✔ Found active emulator! emulator-5554. Will use it
│ error: ⨯ ○ run: COMMAND:

npx react-native run-android --mode=Debug --no-packager --main-activity=SplashActivity --deviceId=emulator-5554

FAILED with ERROR:

Error: Command failed with exit code 1: npx react-native run-android --mode=Debug --no-packager --main-activity=SplashActivity --deviceId=emulator-5554
    at makeError (/Users/gabriele/Desktop/tests/renative/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/Users/gabriele/Desktop/tests/renative/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)


│ Project location: ./platformBuilds/harness_androidtv            │
└─────────────────────────────────────────────────────────────────┘
┌─────────────────────────────────────────────────────────────────┐
│ ✔ SUMMARY | 3/14/2024, 6:11:40 PM | rnv@1.0.0-rc.12             │
│ $ npx rnv run -p androidtv                                      │
├─────────────────────────────────────────────────────────────────┤
│ Project: @rnv/app-harness@1.0.0-rc.12                           │
│ Workspace: rnv                                                  │
│ Platform (-p): androidtv                                        │
│ Engine: engine-rn-tvos                                          │
│ Template: @rnv/template-starter@1.0.0-rc.12                     │
│ App Config (-c): harness                                        │
│ Build Scheme (-s): debug                                        │
│ Target (-t): Television_1080p_API_31                            │
│ Supported Platforms: androidtv                                  │
│ Env Info: darwin | arm64 | node v18.19.0                        │
│ Executed Time: 0h:0m:19s:117ms                                  │
│ info: ✔ Found active emulator! emulator-5554. Will use it
│ error: ⨯ ○ run: COMMAND:

npx react-native run-android --mode=Debug --no-packager --main-activity=SplashActivity --deviceId=emulator-5554

FAILED with ERROR:

Error: Command failed with exit code 1: npx react-native run-android --mode=Debug --no-packager --main-activity=SplashActivity --deviceId=emulator-5554
    at makeError (/Users/gabriele/Desktop/tests/renative/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/Users/gabriele/Desktop/tests/renative/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)


│ error: ⨯ ○ run: COMMAND:

node /Users/gabriele/Desktop/tests/renative/node_modules/react-native-tvos/local-cli/cli.js start --port 8084 --no-interactive --config=metro.config.js

FAILED with ERROR:

Error: Command was killed with SIGTERM (Termination): node /Users/gabriele/Desktop/tests/renative/node_modules/react-native-tvos/local-cli/cli.js start --port 8084 --no-interactive --config=metro.config.js
    at makeError (/Users/gabriele/Desktop/tests/renative/node_modules/execa/lib/error.js:60:11)
    at handlePromise (/Users/gabriele/Desktop/tests/renative/node_modules/execa/index.js:118:26)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)


│ Project location: ./platformBuilds/harness_androidtv            │
└─────────────────────────────────────────────────────────────────┘
➜  app-harness git:(fix/androidManifest_generating)
```

**Review by GabrieleKaceviciute — APPROVED** [review]

**Pull request merged**